manifests/config.pp

```diff
@@ -98,7 +98,7 @@
   }
 
   file { $dnscache::root_servers:
-    source => $dnscache::root_servers_source,
+    content => join($dnscache::root_server_ips, "\n"),
```

Comment:
Is this line accepted on Puppet 3x without the future parser? Not that I particularly care, but is it acceptable on 2.7?